operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -0,0 +1,309 @@
+            throw new Error("There has been an IO error reading the operation definition of the $everything operation.", e);
+        }
+    }
+
+    @Override
+    protected Parameters doInvoke(FHIROperationContext operationContext, Class<? extends Resource> resourceType, String logicalId, String versionId, Parameters parameters, FHIRResourceHelpers resourceHelper) throws FHIROperationException {
+        LOG.entering(this.getClass().getName(), "doInvoke");
+
+        Patient patient = null;
+        try {
+            patient = (Patient) resourceHelper.doRead(PATIENT, logicalId, false, false, null, null);
+        } catch (Exception e) {
+            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("An unexpected error occurred while reading patient " + logicalId, IssueType.EXCEPTION);
+            LOG.throwing(this.getClass().getName(), "doInvoke", exceptionWithIssue);
+            throw exceptionWithIssue;
+        }
+        if (patient == null) {
+            FHIROperationException exceptionWithIssue = buildExceptionWithIssue("Patient with ID " + logicalId + " does not exist.", IssueType.NOT_FOUND);
+            LOG.throwing(this.getClass().getName(), "doInvoke", exceptionWithIssue);
+            throw exceptionWithIssue;
+        }
+
+        Entry patientEntry = buildPatientEntry(operationContext, patient);
+        List<Entry> allEntries = new ArrayList<>(SearchConstants.MAX_PAGE_SIZE);
+        allEntries.add(patientEntry);
+        
+        // We can't always use the "date" query parameter to query by clinical date, only with some resources.
+        // Initial list obtained from the github issue: https://github.com/IBM/FHIR/issues/1044#issuecomment-769788097
+        // Otherwise the search throws an exception. We create a params map with and without and use as needed
+        MultivaluedMap<String, String> queryParameters = parseQueryParameters(parameters);
+        MultivaluedMap<String, String> queryParametersWithoutDates = new MultivaluedHashMap<String, String>(queryParameters);
+        queryParametersWithoutDates.remove(DATE_QUERY_PARAMETER);
+        
+        List<String> resourceTypesOverride = getOverridenIncludedResourceTypes(parameters);
+        List<String> resourceTypes = resourceTypesOverride.isEmpty() ? defaultResourceTypes : resourceTypesOverride;
+        
+        for (String compartmentType : resourceTypes) {
+            MultivaluedMap<String, String> searchParameters = queryParameters;
+            if (!SUPPORT_CLINICAL_DATE_QUERY.contains(compartmentType)) {
+                searchParameters = queryParametersWithoutDates;
+            }
+            Bundle results = null;
+            try {
+                results = resourceHelper.doSearch(compartmentType, PATIENT, logicalId, searchParameters, null, null, null);
+            } catch (Exception e) {
+                LOG.warning("Error retrieving $everything resources of type " + compartmentType + " for patient " + logicalId);
```

Comment:
On this one I figured it would be better to return something than nothing, so that is why I err'd on this side. I can just let the exception go, but I'd like to hear other opinions. 

The product feels more robust/resilient this way, the problem is properly logged and in the event that this happens for say 1 type out of 20 where that 1 type may have only a few instances I think ppl would appreciate at least getting something. If they are really really missing some type they could check the logs and I could add something about using bulkexport instead? I could also print a trace for better problem determination?